docs/release-notes/8.19.asciidoc

```diff
@@ -1,6 +1,35 @@
 [[release-notes-header-8.19.0]]
 == 8.19
 
+[discrete]
+[[release-notes-8.19.5]]
+=== 8.19.5
+
+[discrete]
+[[features-8.19.5]]
+==== New features
+* Adds an {elastic-defend} option to remediate orphaned state by attempting to start Elastic Agent service.
+* Adds more {elastic-defend} Linux diagnostic process `ptrace` events.
+
+[discrete]
+[[enhancements-8.19.5]]
+==== Enhancements
+* Fixes {elastic-defend} error log on Windows where only the first character, usually 'C', was logged instead of a path.
+* Improves reliability and accuracy of reporting of {elastic-endpoint}'s {es} connection.
+
+[discrete]
+[[bug-fixes-8.19.5]]
+==== Fixes
+* Removes `null` in confirmation dialog when bulk editing index patterns for rules ({kibana-pull}236572[#236572]).
+* Fixes the URL passed to detection rule actions via the `{{context.results_link}}` placeholder ({kibana-pull}236067[#236067]).
+* Fixes alert page filtering by checking for empty `dataView` ({kibana-pull}235144[#235144]).
+* Fixes browser fields caching to use the `dataView` ID instead of the index pattern({kibana-pull}234381[#234381]).
+* Adds support in {elastic-defend} for installing eBPF event probes on Linux endpoints when cgroup2 is mounted in a non-standard location or not mounted at all.
+* Adds support in {elastic-defend} for installing eBPF probes on Linux endpoints when taskstats is compiled out of the kernel.
+* Fixes a bug in {elastic-defend} where Linux network events could have source and destination bytes swapped.
+* Removes `.process.thread.capabilities.permitted` and `.process.thread.capabilities.effective` from Linux network events in {elastic-defend}.
+* Fixes a bug in {elastic-defend} where host isolation could auto-release incorrectly. Host isolation now only releases when {elastic-endpoint} becomes orphaned. Intermittent {elastic-agent} connectivity changes no longer alter the host isolation state.
```

Comment:
```suggestion
* Fixes a bug in {elastic-defend} where host isolation could auto-release incorrectly. Host isolation now only releases when {elastic-endpoint} becomes orphaned. Intermittent {elastic-agent} connectivity changes no longer alter the host isolation state.
* Fixes an issue where {elastic-defend} would incorrectly calculate throughput capacity when sending documents to output.  This may have limited event throughput on extremely busy endpoints.
* Fixes an issue in {elastic-defend} installation logging where only the first character of install paths (usually 'C') would be logged.
```